src/lib/Makefile.am

```diff
@@ -14,6 +14,16 @@ lib_LTLIBRARIES = \
 # time.cpp
 # xconnect.cpp
 
+# When releaseing new version of the libraries follow this guide on how to
+# update the version info (-version-info current:revision:age):
+#   1. If the library source code has changed at all since the last update, then increment revision (‘c:r:a’ becomes ‘c:r+1:a’).
+#   2. If any interfaces have been added, removed, or changed since the last update, increment current, and set revision to 0.
+#   3. If any interfaces have been added since the last public release, then increment age.
+#   4. If any interfaces have been removed or changed since the last public release, then set age to 0.
+#
+# Source and more info:
+# https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
+
```

Comment:
Can you document this in some top-level document HOW-TO-RELEASE.md ?

Reply:
I added RELEASE_PROCESS.md . It would be good to document the whole release process there in the future.